sysl2/sysl/parse/utils.go

```diff
@@ -15,13 +15,31 @@ type sourceCtxHelper struct {
 	filename string
 }
 
-func (s sourceCtxHelper) Get(ctx *antlr.BaseParserRuleContext) *sysl.SourceContext {
+func (s sourceCtxHelper) Get(ctx *antlr.BaseParserRuleContext, withText ...bool) *sysl.SourceContext {
```

Comment:
The bool to include extra data makes me feel uneasy because:
1. Reading the source code it's not clear what the trailing `true` is doing.
2. If additional things that can be returned in various combinations are added the code gets messy. 

Perhaps a GetWithText wrapper method?

Reply:
fair enough, will change